userguide.md

```diff
@@ -0,0 +1,50 @@
+# USER GUIDE
+
+## Smaily E-Commerce extension for Magento 2.0+
+>[Smaily](https://smaily.com/) is an intentionally simple tool for sending beautiful email newsletters. 
+
+## Purchase / Download
+
+1. You can purchase **Smaily E-Commerce extension for Magento 2.0+** from [Magento marketplace](https://marketplace.magento.com/smaily-smailyformagento.html) or download it from our [github repository](https://github.com/sendsmaily/smaily-magento-extension).
+
+2. To install our extension follow the [Magento extension installing guide](https://docs.magento.com/marketplace/user_guide/buyers/install-extension.html)
+
+## Account validation
+
+3. To start using Smaily extension navigate to **Stores -> Configuration** section. On the configuration page, find **Smaily Email Marketing and Automation** tab, then click on **Module Configuration**
+
+![Module configuration path](assets/stores_config_module.png)
+
+4. On extension configuration page enter your Smaily API credentials - **subdomain, username, and password**. You must create your API account in Smaily first. You can follow our [tutorial](http://help.smaily.com/en/support/solutions/articles/16000062943-create-api-user) to create one. After that, you can **validate connection** by saving configuration.
+
+![API credentials section](assets/general_settings.png)
+
+## Newsletter subscription form
+
+4. You can **collect newsletter subscribers directly to your Smaily account** using Magento built-in newsletter subscription form.
+
+5. We recommend to **use CATPCHA** to prevent bots from polluting your newsletter subscribers list. You can use two options - **Magento`s text-based CAPTCHA or Google reCAPTCHA**.
+
+![Newletter subscription form](assets/newsletter_subscription_form.png)
+
+## Subscribers synchronization
+
+6. Enable automatic subscribers synchronization feature under **Subscribers synchronization** section. 
+
+7. There is an option to **import additional fields** available from store into Smaily to personalize newsletter emails. **Frequency** can be set from once a week up to once every 4 hours.
```

Comment:
`from once a week up to once every 4 hours` can be a bit vague on which options are available. One could assume that any value between that range is valid. What if we explicitly define what options are available? Would that make sense?

Reply:
Actually I would omit these values. Whoever reads this guide wants to get the first short introduction or overview of what is possible.
> The synchronization **frequency can be adjusted**

Also, the abandoned cart timing option could be shortened.
>  You can **choose timing** when cart is considered abandoned.

Reply:
That might be true for some users, but others would still like to know about the available frequency or timing options.

Reply:
Added the whole list https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2